catalogue_of_life/test/CoLTest.java

```diff
@@ -0,0 +1,70 @@
+package com.vaticle.typedb.example.catalogueOfLife.test;
+
+import com.vaticle.typedb.client.TypeDB;
+import com.vaticle.typedb.client.api.TypeDBClient;
+import com.vaticle.typedb.client.api.TypeDBSession;
+import com.vaticle.typedb.client.api.TypeDBTransaction;
+import com.vaticle.typedb.example.catalogueOfLife.Loader;
+import com.vaticle.typeql.lang.TypeQL;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import static com.vaticle.typeql.lang.TypeQL.var;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class CoLTest {
+    static TypeDBClient client;
+    static TypeDBSession session;
+    static String databaseName = "catalogue-of-life";
+
+    @BeforeClass
+    public static void loadTestData() throws IOException {
+        Path dataDirectory = Path.of("catalogue_of_life/test/data");
```

Comment:
catalogue_of_life/test/data/*.tsv is a much reduced subset of the original dataset for testing purposes.